packages/web/src/components/artist/UserListModal.tsx

```diff
@@ -76,7 +81,16 @@ export const UserListModal = forwardRef<HTMLDivElement, UserListModalProps>(
         ref={ref}
         showDismissButton
         showTitleHeader
-        title={title}
+        title={
+          <div className={styles.titleContainer}>
+            {title === messages.mutuals ? (
+              <IconFollowing className={styles.icon} />
+            ) : (
+              <IconUser className={styles.icon} />
+            )}
+            <span>{title}</span>
+          </div>
+        }
```

Comment:
I don't love having this logic specialcased in UserList - could we have an extra prop that you pass down that gives the option to pass an icon?

Reply:
not sue what you mean. they all have an icon

Reply:
oh you're talking about the modal having an extra prop? I see

Reply:
yeah that seems good. although it will require a change in stems

Reply:
also we did make the modal title type to be ReactNode and not string. so it seems something like this _could_ be expected. although i do see your point